### ISSUE TITLE
Making layer invisible/enabled

### ISSUE THREAD

**Issue by alekbarszczewski** [adobe-photoshop/generator-core#128]
I have following layer structure:

```
group A
   layer A1
   layer A2
   layer A3
```

My goal is to render A+A1, A+A2, A+A3 so each layer separately. But I would like each layer to "extend" to size/bounds of whole group A. And I dont want to use masks. For now I use `minLayerIndex` and `maxLayerIndex` with `enabled/disabled` `layerSettings` param. 

So when I use following:
A1: enabled = true
A2: enabled = false
A3: enabled = false

It renders A1 i bounds of A1. And I want to extend it to have size of all A1 + A2 + A3 bounds. Even when I pass input/output rect, it just ignores it when layer is not enabled.

So, basically I want to enable layer and make it invisible so its bounds include in pixmap and its pixels dont. How I can accomplish this?
